### ISSUE TITLE
Question about taskflow example in readme

### ISSUE THREAD

**Issue by innermous** [cpp-taskflow/cpp-taskflow#23]
I found this quite confusing at first, there's this graph you use as an example of comprehensive taskflow.
![image](https://user-images.githubusercontent.com/8950786/41502204-52cd7cf4-71bd-11e8-8e0d-313f75fde2c8.png)
I might be nitpicking, but isn't S-a1 edge quite redundant here? You can't proceed to a1 without a0 being complete, and that one requires S being finished anyway. 
Correct me if I am wrong here. Thanks.

**Comment by twhuang-uiuc** [comment]
Yes. You are correct. The motivation in this example is to demonstrate how to write this dependency graph using Cpp-Taskflow, regardless of its graph property.

**Comment by innermous** [comment]
@twhuang-uiuc so it was intended, right?

**Comment by twhuang-uiuc** [comment]
Yes.
